app/src/main/res/raw/vn_sdk_config.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Lets go ahead and use this, we can add the actual values to our stash 🙄

We can get Nathan involved in helping us automate internal builds with the SDK demo app

Reply:
Lol. @NFulkerson

Reply:
Our hero.